libsolidity/ast/Types.h

```diff
@@ -593,7 +594,7 @@ class RationalNumberType: public Type
 private:
 	rational m_value;
 
-	/// Bytes type to which the rational can be explicitly converted.
+	/// Bytes type to which the rational can be implicitly converted.
 	/// Empty for all rationals that are not directly parsed from hex literals.
 	Type const* m_compatibleBytesType;
```

Comment:
I think this is an error because this field is actually used in `RationalNumberType::isImplicitlyConvertibleTo()` when deciding if a conversion to fixed bytes is possible.